src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/EnvelopeMsg.java

```diff
@@ -173,6 +178,16 @@ public String toString() {
             .filter(si -> si.getOcrData() != null);
     }
 
+    private String findSubtypeOfScannableItemWithFormType(Envelope envelope) {
+        return envelope
+            .getScannableItems()
+            .stream()
+            .filter(si -> DocumentType.FORM.equals(si.getDocumentType()))
```

Comment:
or `SSCS1` 😢

Reply:
`SSCS1` is the subtype value

Reply:
oh yeah, confused it with `InputaDocumentType`